controls/map/layers/overview.md

```diff
@@ -1,26 +1,27 @@
 ---
 title: Layers Overview
 page_title: .NET MAUI Map Documentation - Map Layers Overview
-description: Check our &quot;Layers Overview&quot; documentation article for Telerik Map for .NET MAUI control.
+description: Learn more about the Map Layers used in Telerik UI for .NET MAUI Map control.
 position: 0
 slug: map-layers-overview
 ---
 
 # .NET MAUI Map Layers
 
-The Map control visualizes spatial data using *layered architecture*. It could display spatial data points (e.g. location of cities), polylines (e.g. road connections), or polygons (shape of countries or continents) in a *geographic coordinate system*. In this coordinate system, a point on the surface of the earth is defined using latitudes and longitudes.
+The Map control visualizes spatial data using *layered architecture*. It can display spatial data points
+(location of cities), polylines (road connections), or polygons (shape of countries or continents) in a *geographic coordinate system*. In this coordinate system, a point on the surface of the earth is defined using latitudes and longitudes.
 
 * **Latitude**: This is measured as the angle from the equatorial plane to the line drawn from the center of the earth to the point on the surface. Usually latitude values range from 90° (toward the North Pole and suffixed with "N") to -90° (towards the South Pole and suffixed with "S").
```

Comment:
- Don't use bold for the terms.
- Use an em dash as a separator for the definition lists.